eng/tox/tox.ini

```diff
@@ -150,4 +156,18 @@ changedir = {toxinidir}
 deps = 
 commands =
     {envbindir}/python {toxinidir}/setup.py --q sdist --format zip -d {distdir}
-    {envbindir}/python {toxinidir}/../../../eng/tox/verify_sdist.py -d {distdir} -t {toxinidir}
\ No newline at end of file
+    {envbindir}/python {toxinidir}/../../../eng/tox/verify_sdist.py -d {distdir} -t {toxinidir}
+
+
+[testenv:devtest]
+pre-deps =
+  wheel
+deps = {[devtools]deps}
+changedir = {toxinidir}
+commands = 
+    {envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py -d {distdir} -p {toxinidir} \
```

Comment:
Nitpick. If we're going to split commands can we have it all of the options on a separate line? ref line 118 for what I mean.